justfile

```diff
@@ -15,7 +15,7 @@ promote-to-standard CHAIN:
 
 # Generate auto-generated files
 codegen: clean-add-chain
-	go run superchain/internal/codegen/main.go
+	CODEGEN=true go run superchain/internal/codegen/main.go
```

Comment:
Curious about why this is needed?

Reply:
Ah, so since the codegen is run in circleci, the configured rpcs are used instead of the default ones in the superchain init when the `Superchains` list is created. Since that list is used in the configs.toml generation to get the `SuperchainConfig` object (containing the rpcs), those rpcs are different when run locally without the [circle ci rpc envs](https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/init.go#L76-L77) set and remotely in circleci environment.